Makefile.am

```diff
@@ -185,4 +188,4 @@ if BUILD_RGMANAGER
 endif
 
 clean-generic:
-	rm -rf $(SPEC) $(TARFILES)
+	rm -rf $(pkgconfig_DATA) $(SPEC) $(TARFILES)
```

Comment:
You shouldn´t clean pkgconfig_DATA here. Since it´s a configuration file (according to AC_CONF_FILES) it will be cleaned only at make maintainer-clean (automatically). The only way to regenerate that file is via ./configure otherwise and that would break make all, make clean, make install.